src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/AutoSuggestBoxTests/AutoSuggestBox_PopupFocus.xaml

```diff
@@ -0,0 +1,38 @@
+﻿<Page
+    x:Class="UITests.Windows_UI_Xaml_Controls.AutoSuggestBoxTests.AutoSuggestBox_PopupFocus"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UITests.Windows_UI_Xaml_Controls.AutoSuggestBoxTests"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+	<StackPanel>
+		<Button Content="Click me">
+			<Button.Flyout>
+				<Flyout>
+					<StackPanel>
+						<TextBlock>
+							Repro Steps:
```

Comment:
I think `InputInjector` now works on WASM in runtime tests - so theoretically this should be possible to turn into a runtime test as well, or not?